tools/convert/README.md

```diff
@@ -579,7 +729,7 @@ These might be useful to keep beside a set of multi-gigabyte binary traces to
 allow fast grepping when looking for traces that use particular named arguments.
 
 ```bash
-gfxrecon-convert --output stdout vkcube.f1.gfxr | jq  -c "[.. | objects | keys[]] | unique" | sed "s/\"args\",//" | sed "s/\"index\",//" | sed "s/\"name\",//" | sed "s/\"return\",//" | sed "s/,\"vkFunc\"//" | sort | uniq
+gfxrecon-convert --output stdout vkcube.f1.gfxr | jq  -c "[.. | objects | keys[]] | unique" | sed "s/\"args\",//" | sed "s/\"index\",//" | sed "s/\"name\",//" | sed "s/\"return\",//" | sed "s/,\"function\"//" | sort | uniq
```

Comment:
I think this example needs `--format jsonl`.  I didn't check any further; could you verify whether the examples need `--format jsonl` or not?